Jenkinsfile

```diff
@@ -1,6 +1,5 @@
 // Builds a module using https://github.com/jenkins-infra/pipeline-library
 buildPlugin(useContainerAgent: true, configurations: [
-    [platform: 'linux', jdk: 11], 
-    [platform: 'windows', jdk: 11],
-    [platform: 'linux', jdk: 17]
+    [platform: 'linux', jdk: 21],
+    [platform: 'windows', jdk: 17],
```

Comment:
Latest recommendation from [the archetype](https://github.com/jenkinsci/archetypes).